cogs/sniper.py

```diff
@@ -62,19 +68,19 @@ async def on_reaction_remove(self, reaction, user):
                                       extras=reaction.message.jump_url)
 
     @commands.command()
-    async def snipe(self, ctx):
+    async def snipe(self, message):
```

Comment:
Had to do it a bit different because we call these from the easter egg listener which doesn't have a context object to pass, so i used an extra argument, let me know if you had a better way in mind?